### PR TITLE
Adds problem solving session event

### PR DESCRIPTION
**Pull request by nivasini**

### CODE REVIEW

File: content/W2016/W16-event-problem-solving-session.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+title: Problem Solving Session
+Date: 2016-02-01
+Category: Events
+Tags: problem solving, computing, math
+Slug: problem-solving-w16
+Author: Nivasini Ananthakrishnan
+Summary: Evening of math and coding
+
+Thanks for coming out to the first problem solving session for this semester! We hope you had fun socializing through math and computing problems. If you missed it you can find the problems [here](https://drive.google.com/folderview?id=0B68pQQep_ETbeVg1NUUtUDdRNnc&usp=sharing). Or come visit us during our office hours for copies of the problem sets.
```
Comment (fboxwala) [inline content/W2016/W16-event-problem-solving-session.md]: So even though its already been in the past, you should make the event post in the present tense, as all the rest of them are. I know that seems weird but it makes things more consistent

File: content/W2016/W16-event-problem-solving-session.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+
+## Event Details ##
+
++ **Who:** Women and Non-binary individuals 
++ **What:** Problem Solving Session
++ **Where:** M3
```
Comment (fboxwala) [inline content/W2016/W16-event-problem-solving-session.md]: you should include the room number asw

File: content/W2016/W16-event-problem-solving-session.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+title: Problem Solving Session
+Date: 2016-02-01
+Category: Events
+Tags: problem solving, computing, math
+Slug: problem-solving-w16
+Author: Nivasini Ananthakrishnan
+Summary: Evening of math and coding
```
Comment (fboxwala) [inline content/W2016/W16-event-problem-solving-session.md]: This is usually a full sentence, so maybe add "an" to the beginning of it?

**Comment by fboxwala** [comment]
Nice job cleaning up the commit history! This is much much better :)

Once you edit the file with the small changes I made just do:

`git add [filename]`
`git commit --amend`
(this will open a text editor, just save the file with no changes)
`git push -f origin HEAD`

**Comment by ehashman** [comment]
+1 to seeing this as a separate commit.

**Comment by fboxwala** [comment]
lgtm

**Pull request merged**